content/dev/features/dataset-folders.md

```diff
@@ -1,48 +1,45 @@
+{{< figure src="dev/features/images/folders-before-and-after.png" class="img-fluid">}}
 
-**Personal** will only appear to users who have permission to import data. It contains any datasets you have imported and not moved to another folder.
+Click on a folder's name to enter it. When you do, you'll see at the top of the screen the folder you are currently in and the path to that folder.
 
-**Shared with Me** contains datasets that have been shared with you directly without access being granted to the folder that contains it. If no datasets have been shared with you, this folder will be hidden.
+{{< figure src="dev/features/images/folders-breadcrumb.png" class="img-fluid">}}
 
-If you have access to the folder that contains the dataset, it will appear in that location.
+To jump back to a higher folder, just click on it in the header.
 
-As always, when you’re not sure where to find a dataset we highly recommend using the search bar at the top of screen. Our search allows you to filter e.g. just dataset names.
+Within a folder, you can sort the list of datasets and folders by clicking the column headers at the top of the list. By right-clicking on the header, you can customize which columns appear in the list, so you can sort the view by most-recently accessed, or alphabetical, or whatever you want.
 
-#### 3. The header shows your current folder and how you got there
+The other major change to navigation involves the "Personal Project".
+Previously, any datasets that were shared with you, as well as any datasets that you had imported yourself, were bundled together in the personal project. This was confusing to some, and it meant that some users' personal projects were full of lots of data that belonged elsewhere.
 
-{{< figure src="dev/features/images/folders-breadcrumb.png" class="img-fluid">}}
-At the top of the screen, the header now shows which folder you are currently in and the path to that folder. To jump back to a higher folder, just click on it.
+We've simplified where datasets are presented in folders to you. If you have access to a folder, you'll see that listed at the top level, and everything contained inside that folder will show up inside that folder. **Personal** now contains any datasets you have imported and not moved to another folder. Only users who have permission to import data will see Personal. The other special folder, **Shared with Me**, contains datasets that have been shared with you without access being granted to the folder that contains it. If no datasets have been shared with you in this way, this folder won't appear.
+
+As always, when you’re not sure where to find a dataset, use the search bar at the top of screen. If you're looking for a specific dataset by name, you can narrow your search criteria to match only on dataset names.
+
+For more details on navigating dataset folders, see [the support page](http://support.crunch.io/articles/cEBBbgki/Navigating-Datasets-Early-Access) for more information.
```

Comment:
These links will need to be updated, though the timing of that isn't critical.

Reply:
I think in order to change the url I'll need to basically copy the card. I've already updated the content to remove the early access material and title.